src/fosslight_scanner/fosslight_scanner.py

```
@@ -553,20 +577,29 @@ def run_main(mode_list, path_arg, dep_arguments, output_file_or_dir, file_format
-                        for sub_item in os.listdir(src_item):
-                            shutil.move(os.path.join(src_item, sub_item), os.path.join(dst_item, sub_item))
-                    else:
-                        shutil.move(src_item, dst_item)
-                shutil.rmtree(output_path)
+                    try:
+                        if os.path.isdir(src_item) and os.path.exists(dst_item):
+                            for sub_item in os.listdir(src_item):
+                                shutil.move(os.path.join(src_item, sub_item), os.path.join(dst_item, sub_item))
+                        else:
+                            shutil.move(src_item, dst_item)
+                    except Exception as ex:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)